src/api/index.js

```diff
@@ -1,8 +1,9 @@
-const express = require("express");
+const express = require('express');
 
-const { healthRouter } = require('../routes/health/health.router')
```

Comment:
Just curious, was this code autogenerated by Express?

Reply:
That's part of the boilerplate code from the `be-cohort1-sample` :)

Reply:
Gotcha :)